controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -118,13 +118,15 @@ controls:
     controls:
     - id: 5.4.1
       title: Prefer using secrets as files over secrets as environment variables
-      status: pending
-      rules: []
+      status: partial
```

Comment:
why are these partial? Did you mean manual?

Reply:
Updated to manual since these rules don't automate anything in the operator.